spec/features/step_by_step_actions_spec.rb

```diff
@@ -21,11 +21,31 @@
     end
   end
 
+  context "Step by step has been submitted for 2i" do
+    scenario "show the relevant actions to the step by step author" do
+      given_there_is_a_step_by_step_with_status "submitted_for_2i"
```

Comment:
I know I wrote the original pseudocode for this, but am having second thoughts about the need to pass a parameter. I think we should just use `given_there_is_a_step_by_step_that_has_been_submitted_for_2i`, adding it to `step_nav_steps.rb` in the same style as the others (i.e. taking lines 35-38, renaming it and moving it).

If it starts to become hard to understand exactly what the status is from the method name alone, we can move to this parameterised approach in future.

Reply:
Sure, update. It makes sense to live next to the other helpers which provide us with a step-by-step in a certain state.